src/shared/shared.module.ts

```diff
@@ -21,8 +21,8 @@ import { AllExceptionsFilter } from './filter/all-exceptions.filter';
         database: configService.get<string>('database.name'),
```

Comment:
I guess we no longer need this since we are defining the `ormconfig.json` file.
> Alternatively, rather than passing a configuration object to forRoot(), we can create an ormconfig.json file in the project root directory.

Reply:
We can try to unify this ormconfig, I'll create an issue